src/libraries/System.Resources.Extensions/tests/BinaryFormatTests/Common/EventOrderTests.cs

```diff
@@ -0,0 +1,1023 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.Serialization;
+using FormatTests.Common.TestTypes;
+using ValueType = FormatTests.Common.TestTypes.ValueType;
+
+namespace FormatTests.Common;
```

Comment:
Existing tests in this library use the root namespace `System.Resources.Extensions.Tests`.  Be consistent with the library you're merging into.

Reply:
I still see this file as being in the namespace `FormatTests.Common`, but the issue was marked as resolved with no comment... so, reactivating.

Reply:
PEBCAK.  I ended up in a commit-limited view when I thought I was in live/"all commits".